endpoints/openrtb2/sample-requests/amp/valid-supplementary/aliased-buyeruids-case-insensitive.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Could you also add a test using a bidder name where it's original formatting isn't already lowercase (i.e. yahooAds)

Reply:
Added in [8b7a55b](https://github.com/prebid/prebid-server/pull/3123/commits/8b7a55b3c066d5272be45bd29ca23ed3bf14fc75)